### ISSUE TITLE
The axis grid line gets incontinious after deselect the coordinate axes checkbox

### ISSUE THREAD

**Issue by PennyQ** [glue-viz/glue-vispy-viewers#121]
I found the grid line of the axis becomes incontinious after deselect the `coordinate axes` checkbox and select it again, it looks obvious when rotating as shown below:
![image](https://cloud.githubusercontent.com/assets/13411839/14662013/6ac93726-0680-11e6-80de-387b652fd5fa.png)

**Comment by astrofrog** [comment]
Yeah, I noticed that too - this is in the volume viewer, correct?

**Comment by PennyQ** [comment]
Yeah, scatter plot is ok
